core/src/test/java/io/questdb/cairo/GeoHashesTest.java

```diff
@@ -697,9 +696,8 @@ private void testUnsafeFromStringTruncatingNl(CharSequence token, BiFunction<Lon
         final long lo = Unsafe.malloc(len, MemoryTag.NATIVE_DEFAULT);
         final long hi = lo + len;
         try {
-            sun.misc.Unsafe unsafe = Unsafe.getUnsafe();
             for (long p = lo; p < hi; p++) {
-                unsafe.putByte(p, (byte) token.charAt((int) (p - lo)));
+                Unsafe.getUnsafe().putByte(p, (byte) token.charAt((int) (p - lo)));
```

Comment:
why access it in the loop and not as before, I am genuinely interested, there must be a reason I fail to see.